app/model/ClientConfig.scala

```diff
@@ -0,0 +1,15 @@
+package model
+
+import play.api.libs.functional.syntax._
+import play.api.libs.json.{JsValue, Json, JsPath, Format}
+
+case class ClientConfig(capiUrl: String, capiKey: String, tagTypes: List[String])
+
+object ClientConfig {
+
+  implicit val clientConfigFormat: Format[ClientConfig] = (
```

Comment:
I don't really get what this is doing, or alternatively why it is being done this way.